### DISCUSSION TITLE
night/day mode

### DISCUSSION THREAD

**Discussion by publine2** [dail8859/NotepadNext#505] [Ideas]
Would be good to have a night/day mode (dark and bright interface colors).

**Comment by jonchines** [comment]
I concur.  After testing NotepadNext's large file performance, this was the second thing I looked for.  Theming or light/dark (at a minimum) would be great.  Even if it wasn't a UI button/selection but some property or file that could be modified.

**Comment by aCtionBR** [comment]
I support that. This was the first setting I was looking for after installed it. I am using it on Linux.

**Comment by DucksLive** [comment]
+1 for even a basic dark theme. Nice work here, thank you.  :)

**Comment by AdamRealE** [comment]
There is a 2 year old PR for this that has been worked extensively but never merged:
https://github.com/dail8859/NotepadNext/pull/411

**Comment by WKenX** [comment]
Maybe prioritize that feature? Modify it via config file.
